main.go

```diff
@@ -12,6 +12,7 @@ func main() {
 	router := mux.NewRouter()
 
 	router.HandleFunc("/biz/validate", getBizYelp).Methods("POST")
+	router.HandleFunc("/biz/register", registerBiz).Methods("POST")
```

Comment:
Is this a new registered route for registration? Creates a user?

Reply:
This is Business Partner Registration after they press Yelp validate.